spyder/app/tour.py

```diff
@@ -94,6 +99,7 @@ def get_tour(index):
     """
     sw = SpyderWidgets
     qtconsole_link = "https://qtconsole.readthedocs.io/en/stable/index.html"
+    ending_link = "https://docs.spyder-ide.org/current/index.html"

```

Comment:
We should add `__doc_url__ = https://docs.spyder-ide.org/` to `spyder/__init__.py` and reference it here and in the relevant menu item in MainWindow (and use it as the base URL for other links to the docs), instead of scattering multiple references to it throughout the code, to avoid bugs, keep things clean and easy to follow, and make it much simpler to change in the future.